### PR TITLE
Fix header/footer + add button to TextSection + Fix Hero

### PR DESCRIPTION
**Pull request by AnneSophieO**
Reusing flexible link structure from C4
Reusing Hero (ish) from C4

### CODE REVIEW

File: src/components/sections/Hero/index.tsx [hunk 1]
```diff
@@ -1,22 +1,21 @@
 import BaseImage from '../../utilities/BaseImage'
 import type { THero } from '@/sanity/queries/objectQueries/hero'
 
-type Props = {
-  data: THero
-}
-
-export default function Hero({ data }: Props) {
+export default function Hero(props: THero) {
```
Comment (kasperfilstrup) [inline src/components/sections/Hero/index.tsx:4]: Much better👏

File: src/components/utilities/BaseImage/index.tsx [hunk 2]
```diff
@@ -42,7 +43,7 @@ export default function BaseImage({
     <picture>
       {aspectRatioDesktop && <Source media='(min-width: 640px)' ratio={aspectRatioDesktop} />}
       <Image
-        className={className + 'sm:bg-black'}
+        className={cn('w-full', className)}
```
Comment (kasperfilstrup) [inline src/components/utilities/BaseImage/index.tsx:46]: What happens here? no more bg-black on small, or?

File: src/sanity/queries/helperQueries/referencePath.ts
Comment (kasperfilstrup) [inline src/sanity/queries/helperQueries/referencePath.ts:1]: Won't this destroy the URL hiearchy if pages are nested? Do you know @mjthias ?

**Review by kasperfilstrup — APPROVED** [review]
Very nice work, A.

**Review by mjthias — COMMENTED** [review]
Godt arbejde

**Pull request merged**